src/ui/public/vislib/vis.js

```diff
@@ -64,8 +64,12 @@ export default function VisFactory(Private) {
         uiState.on('change', this._uiStateChangeHandler = () => this.render(this.data, this.uiState));
       }
 
+      this.resizeChecker.stopSchedule();
```

Comment:
what do you think about adding function to the vislib class ? (take a look at latest commit) ?